extension/php_meminfo.h

```diff
@@ -1,20 +1,39 @@
 #ifndef PHP_MEMINFO_H
 #define PHP_MEMINFO_H 1
 
+#include "php.h"
+
 extern zend_module_entry meminfo_module_entry;
 #define phpext_meminfo_ptr &meminfo_module_entry
 
 #define MEMINFO_NAME "PHP Meminfo"
 #define MEMINFO_VERSION "2.0.0-beta1"
 #define MEMINFO_AUTHOR "Benoit Jacquemont"
-#define MEMINFO_COPYRIGHT  "Copyright (c) 2010-2021 by Benoit Jacquemont & contributors"
+#define MEMINFO_COPYRIGHT "Copyright (c) 2010-2021 by Benoit Jacquemont & contributors"
 #define MEMINFO_COPYRIGHT_SHORT "Copyright (c) 2010-2021"
 
+ZEND_BEGIN_MODULE_GLOBALS(meminfo)
+    zend_bool dump_on_limit;
+ZEND_END_MODULE_GLOBALS(meminfo)
+
+static ZEND_DECLARE_MODULE_GLOBALS(meminfo)
+#define MEMINFO_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(meminfo, v)
+
 PHP_FUNCTION(meminfo_dump);
+PHP_MSHUTDOWN_FUNCTION(meminfo);
+PHP_MINIT_FUNCTION(meminfo);
+PHP_MINFO_FUNCTION(meminfo);
+PHP_GINIT_FUNCTION(meminfo);
+
+PHP_INI_BEGIN()
+STD_PHP_INI_ENTRY("meminfo.dump_on_limit", "Off", PHP_INI_ALL, OnUpdateBool, dump_on_limit, zend_meminfo_globals, meminfo_globals)
+PHP_INI_ENTRY("meminfo.dump_dir", "/tmp", PHP_INI_ALL, NULL)
```

Comment:
@BitOne Do you think we can just let this default to "/tmp" for now? It might not be the best for Windows, but I think most users will probably set the dump_dir if they are going to use this anyway right?